test/TestUtilities/Test.Utility/Signing/ChainCertificateRequest.cs

```diff
@@ -14,8 +14,8 @@ public class ChainCertificateRequest
 
         public bool IsCA { get; set; }
 
-        public bool ConfigureCrl { get; set; } = true;
+        public bool ConfigureCrl { get; set; } = false;
```

Comment:
The [`default`](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/default-values-table) value of bool is false.

Reply:
Thanks~Changed accordingly.